examples/BeerDistributionProblem_resolve.py

```diff
@@ -68,10 +68,10 @@
 
 # The problem data is written to an .lp file
 prob.writeLP("BeerDistributionProblem.lp")
-for demand in range(500, 601, 10):
+for demand_value in range(500, 601, 10):
```

Comment:
same as with `_dict`, is this some kind of naming convention based on the type of the object?

Reply:
Similar to previously there is already a `demand` in scope and this was me fixing the collision.